src/main/java/com/google/step/snippet/servlets/SearchServlet.java

```diff
@@ -40,6 +46,12 @@
   private static final String GEEKS_CSE_ID = "INSERT_GEEKSFORGEEKS_CSE_ID";
   private static final String API_KEY = "INSERT_API_KEY";
   private static final String CSE_URL = "https://www.googleapis.com/customsearch/v1";
+  private static final List<Client> clients =
```

Comment:
I don't think this should be `static` - although the clients don't currently contain internal state, I think of the clients as belonging to the instance of SearchServlet, not the class itself. @joanchenx if you have thoughts

Reply:
Yeah +1 it feels like the clients should be a part of the SearchServlet instance